pom.xml

```diff
@@ -33,7 +33,7 @@
     </parent>
     <groupId>org.jenkins-ci.plugins.workflow</groupId>
     <artifactId>workflow-api</artifactId>
-    <version>2.1-SNAPSHOT</version>
+    <version>2.2.blockapis-SNAPSHOT</version>
```

Comment:
Temporary versioning to allow use in stage view/pipeline graph analysis.  This will be changed before release.